packages/aws-cdk-lib/custom-resources/lib/provider-framework/provider.ts

```diff
@@ -250,6 +250,20 @@ export class Provider extends Construct implements ICustomResourceProvider {
     };
   }
 
+  private addPermissions(frameworkLambda: lambda.Function, arnOfUserDefinedHandlerLambda: lambda.IFunction) {
```

Comment:
nit: the name `arnOfUserDefinedHandlerLambda` is miss leading, I believe it should be `userDefinedHandlerLambda`